src/extension/prompts/node/agent/summarizedConversationHistory.tsx

```diff
@@ -429,27 +499,11 @@ class ConversationHistorySummarizer {
 
 		let summaryResponse: ChatResponse;
 		try {
-			const toolOpts = mode === SummaryMode.Full ? {
-				tool_choice: 'none' as const,
-				tools: normalizeToolSchema(
-					endpoint.family,
-					this.props.tools?.map(tool => ({
-						function:
-						{
-							name: tool.name,
-							description: tool.description,
-							parameters: tool.inputSchema && Object.keys(tool.inputSchema).length ? tool.inputSchema : undefined
-						}, type: 'function'
-					})),
-					(tool, rule) => {
-						this.logService.logger.warn(`Tool ${tool} failed validation: ${rule}`);
-					},
-				),
-			} : undefined;
+			// Don't include tools in the summarization request to prevent tool calling
 			summaryResponse = await endpoint.makeChatRequest('summarizeConversationHistory', ToolCallingLoop.stripInternalToolCallIds(summarizationPrompt), undefined, this.token ?? CancellationToken.None, ChatLocation.Other, undefined, {
 				temperature: 0,
 				stream: false,
-				...toolOpts
+				tool_choice: 'none' // Explicitly disable tool calling during summarization
```

Comment:
Tools must be provided to use tool_choice- I believe some provider errors when we do thsi